ra-tls/Makefile

```diff
@@ -37,13 +37,13 @@ CC ?= gcc
 
 export DEBUG PREFIX BINDIR LIBDIR INCDIR SGX_SDK SGX_RA_TLS WOLFSSL CC
 
-deps := $(LIBDIR)/libwolfssl.sgx.static.lib.a $(LIBDIR)/libsgx_ra_tls_wolfssl.a $(LIBDIR)/libcurl-wolfssl.a
+deps := $(LIBDIR)/libwolfssl.sgx.static.lib.a $(LIBDIR)/libra-tls.so $(LIBDIR)/libsgx_ra_tls_wolfssl.a $(LIBDIR)/libra-tls-sgx.so $(LIBDIR)/libcurl-wolfssl.a
 all: $(deps) $(BINDIR)/elv $(BINDIR)/ra-tls-server $(BINDIR)/Wolfssl_Enclave.signed.so
 
-$(BINDIR)/elv: $(BINDIR) $(LIBDIR)/libra-challenger.a $(LIBDIR)/libwolfssl.a
+$(BINDIR)/elv: $(BINDIR) $(LIBDIR)/libra-tls.so $(LIBDIR)/libcurl-wolfssl.a $(LIBDIR)/libwolfssl.a
```

Comment:
why we still need libcurl-wolfssl.a and libwolfssl.a? I believe all untrusted codes should go a .so.

Reply:
I have removed libcurl-wolfssl.a and libwolfssl.a when build elv and ra-tls-server